README.md

```diff
@@ -79,6 +84,79 @@ console.log(fittedParams);
  */
 ```
 
+For data with and combination of signals with shapes between gaussian and lorentzians, we could use the kind pseudovoigt to fit the data.
+
+```js
+import { optimizeSum } from 'ml-spectra-fitting';
+import { SpectrumGenerator } from 'spectrum-generator';
+
+const generator = new SpectrumGenerator({
```

Comment:
Try to use the function with an array of peaks and default parameters to reduce the length of this example